# sets maximal velocity and acceleration values along the trajectory
dynamic_params: {
  max_v: 5.0,
  max_a: 2.0
}
 
waypoints: {
  vertices: [   
    #{ position: [ -38.0,  10.0,   6.5] }, # Initial Pose
    { position: [ -50.0,   0.0,  10.0], velocity: [ 0.0, 0.0, 0.0], acceleration: [ 0.0, 0.0,  0.0] }, # First light 
    { position: [ -73.0, -10.0,  10.0], velocity: [ 3.0, 0.0, 0.0], acceleration: [ 0.0, 0.0,  0.0] }, # Avoid Tree
    { position: [-100.0, -10.0,  10.0], velocity: [ 3.0, 0.0, 0.0], acceleration: [ 1.0, 0.0,  0.0] }, # WP1 towards entrance
    { position: [-150.0, -10.0,  10.0], velocity: [ 3.0, 0.0, 0.0], acceleration: [ 0.0, 0.0,  0.0] }, # WP2 towards entrance
    { position: [-200.0, -5.0,  12.0], velocity: [ 3.0, 0.0, 0.0], acceleration: [ 0.0, 0.0,  0.0] }, # WP3 towards entrance  
    { position: [-320.0,   15.0,  15.0], velocity: [ 0.0, 0.0, 0.0], acceleration: [ 0.0, 0.0,  0.0] }, # Cave Entrance
  ]
}
 
